draft-ietf-quic-transport.md

```diff
@@ -888,6 +888,9 @@ that are encrypted with a key it has not yet computed. Clients MAY drop these
 packets, or MAY buffer them in anticipation of later packets that allow it to
 compute the key.
 
+If a client receives a packet that has an unsupported version, it MUST discard
+that packet.
+
 
```

Comment:
Isn't that an invariant? Otherwise the next version could require the client to do a best effort on unknown versions, and a client would not know what guides the world.